ubuntudesign/docs.vanillaframework.io
# Improve the table of contents

## Done
Update to the latest docs release v1.6.0. Add the separater class to the Back to top list item. Updated the JS to have the true value so fixed scrolling is smoother.

## QA
- Run `build-html`
- Run `caddy`
- Go to 127.0.0.1:8543/en/base/forms
- Check the table of contents is styled with space
- Scroll down and see that the sticky functionality is smooth

## Details
Fixes https://github.com/vanilla-framework/vanilla-docs-theme/issues/57
